packages/internal/src/utils.js

```diff
@@ -266,10 +266,10 @@ harden(bindAllMethods);
  * A more constrained version of {deeplyFulfilled} for type safety until https://github.com/endojs/endo/issues/1257
  * Useful in starting contracts that need all terms to be fulfilled in order to be durable.
  *
- * @type {<T extends {}>(unfulfilledTerms: T) => import('@endo/eventual-send').ERef<DeeplyAwaited<T>>}
+ * @type {<T extends {}>(unfulfilledTerms: T) => Promise<DeeplyAwaited<T>>}
  */
 export const deeplyFulfilledObject = obj => {
-  assert(isObject(obj), 'param must be an object');
+  isObject(obj) || Fail`param must be an object`;
```

Comment:
I also took the opportunity to normalize this assertion into our dominant style. But this change is a drive by. It has nothing to do with the purpose of this PR. Let me know if you'd like it migrated to a separate PR.

Reply:
The only downsides I know to drive-bys are,
1. the git blame commit message is inaccurate
2. the change may not get the proper reviewers

In this case the change has sufficient review, so no need for a separate PR. You could solve the commit message by just having a second commit in the same PR, but I don't think it's worth the hassle.

Reply:
Will remember this advise for future, thanks. Resolving.